Makefile

```diff
@@ -23,4 +27,16 @@ vendor:
 test: vendor
 	go test ./pkg/...
 
+images: asb-image
+
+asb-image:
```

Comment:
How about changing the name to ```ansible-container```?  This is specific to ansible-container and later we'll merge the PR that adds building without ansible-container.  That way this recipe reflects how the apb is being built.